### DISCUSSION TITLE
How to stream mpd accross the internet

### DISCUSSION THREAD

**Discussion by Celibistrial** [MusicPlayerDaemon/MPD#1613] [Q&A]
So I have an "httpd" audio output setup, which works pretty well over lan 
how would I go about making this stream accessible from the internet?
probably a really dumb idea tbh, but it could be fun to have a URL you could connect to from anywhere and instantly have your music play

**Comment by MaxKellermann** [comment]
You need to make that port MPD binds to accessible on a public IP address. That isn't really a MPD question, but a networking question.

**Reply by Celibistrial** [reply]
I see , how safe would that be?

**Reply by MaxKellermann** [reply]
Dunno, what do you fear?

**Reply by Celibistrial** [reply]
idk, so how would I go about doing this? (I do realise I should probably not ask here)
would just port forwarding the MPD ports work?

**Reply by MaxKellermann** [reply]
That would work.

**Reply by Celibistrial** [reply]
Ah thanks, the http stream works. I also exposed localhost::6600 but I can't control MPD from the public URL using MPDroid , it refuses to connect for some reason
